.github/workflows/python-build-and-test.yml

```diff
@@ -2,20 +2,77 @@ name: Python unit tests
 
 on: [push]
 
+env:
+  PYTHONPATH: ${{ github.workspace }}
+
 jobs:
-  build:
+  build-and-test-1:
+    runs-on: ubuntu-latest
+    container: quarkchaindocker/pyquarkchain:mainnet1.1.1
 
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Project Dependencies
+      env:
```

Comment:
a side note, can save some lines by removing `env` but have line 18 to be `PYTHONPATH=/code/pyquarkchain pip3 install -e .`